### PR TITLE
Add vertical bars

### PR DESCRIPTION
**Pull request by rbrtsmith**
* Need to add tests
* Not sure how we configure the API so the consumer decides which row on the timebar is used for the vertical grid lines?  I am just manually selecting the months here.
screen shot 2017-06-08 at 16 16 13

### CODE REVIEW

File: src/components/Timeline/index.jsx [hunk 1]
```diff
@@ -70,7 +71,9 @@ class Timeline extends Component {
           width={time.timelineWidth}
           sticky={sticky}
         />
-        <Body time={time} tracks={tracks} clickElement={clickElement} />
+        <div style={buildGridStyle(time, timebar[2])}>
```
Comment (WebSeed) [inline src/components/Timeline/index.jsx]: I'd put inside `<Body />`

File: src/utils/buildGridStyle.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const getStartXs = (time, { cells }) => cells.map(cell => time.toX(cell.start))
+
+const buildGridStyle = (time, timebar) => {
+  const gradient = getStartXs(time, timebar).reduce((str, x) =>
+    `${str} white ${x}px, #eee ${x}px, #eee ${x + 1}px, white ${x + 1}px,`
+  , '').slice(0, -1)
+  return { background: `linear-gradient(to right, ${gradient})` }
```
Comment (WebSeed) [inline src/utils/buildGridStyle.js]: Not sure about doing this as `linear-gradient` background - not sure if better to have DOM elements (as per timebar cells).

**Comment by rbrtsmith** [comment]
Playing devils advocate: Any reason not to use gradients?  I would have
thought perf would be better using fewer DOM nodes.

**Comment by WebSeed** [comment]
Just wonder about very large timelines - you have to render a gradient across a very large area which I imagine is expensive. I also wonder if you will get some weird off by one / sub-pixel glitches given timebar is DOM rendered? DOM leaves opportunities to optimise what is rendered based on what's visible in the viewport (not something lib does yet but browser may optimise on outlet behalf). It's also more consistent with how we have approached the timebar.

**Comment by rbrtsmith** [comment]
I'm not convinced, if we went with the same approach as the header we are repeatedly using box shadow on each node which is as we know not the most performant property.

I will however create another branch taking this approach then we can compare the two.

**Comment by WebSeed** [comment]
Use of box-shadow is neither here nor there I think. Could be swapped out for border if that's an issue and as previously discussed I think concern is mainly around large shadows. I just don't think having a CSS background with 100s and potentially 1000s of entries gives us, or the browser, room to optimise i.e. not sure it's scalable.

**Pull request closed without merging**